fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml

```diff
@@ -1,5 +1,6 @@
 # skip boilerplate check
 
+name: dev-dataplatform
```

Comment:
are you intentionally removing the region suffix?

Reply:
Yes. We have added support in the factory a while ago, and it makes the code a lot more readable. Compare for example the ncc stage, where subnet references were some of the longest and ugliest I've ever seen in FAT/Fabric. It also makes the code more portable.